.github/workflows/ci.yml

```diff
@@ -68,7 +68,9 @@ jobs:
                     | tee docker-bake-template-meta.json
             - name: Build images with buildx bake
               id: bake
-              uses: docker/bake-action@v2.3.0
+              env:
+                  BUILDX_NO_DEFAULT_ATTESTATIONS: 1
```

Comment:
This is not work actually, for the `docker/bake-action` the correct way to disable [attestation](https://docs.docker.com/build/attestations/slsa-provenance/) is as shown in https://github.com/docker/bake-action#inputs.
The attestation need to be disabled otherwise there will be an `unknown/unknown` image in the packages registry https://github.com/aiidalab/aiidalab-docker-stack/pkgs/container/base-with-services.